### PR TITLE
GH-1603 Fix plugin version dropdown color in dark theme

### PR DESCRIPTION
**Pull request by sudhinm**
#1603

### CODE REVIEW

File: reposilite-site/pages/plugin/[id].js [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+import {MDXRemote} from "next-mdx-remote"
+import {ChevronRightIcon} from "@chakra-ui/icons"
 import Head from "next/head"
-import { useEffect, useState } from "react"
-import xml2js from 'xml2js'
-import { ColorModeStyles, useColorModeValue } from "nextjs-color-mode"
-import { chakraColor } from "../../helpers/chakra-theme"
+import {useEffect, useState} from "react"
+import {ColorModeStyles, useColorModeValue} from "nextjs-color-mode"
+import {chakraColor} from "../../helpers/chakra-theme"
```
Comment (dzikoysk) [inline reposilite-site/pages/plugin/[id].js:23]: Revert those changes to formatting to keep it consistent with the rest of files

**Review by dzikoysk — CHANGES_REQUESTED** [review]
Thanks! I'll merge it after a fix to imports section :)

File: reposilite-site/pages/plugin/[id].js (hunk 1 above)
Comment (sudhinm) [reply reposilite-site/pages/plugin/[id].js:23]: sure @dzikoysk , will do the changes.

**Comment by sudhinm** [comment]
@dzikoysk Updated the changes requested. Thank you!

**Review by dzikoysk — APPROVED** [review]

**Comment by dzikoysk** [comment]
Thanks!

**Pull request merged**